api/SVGFilterPrimitiveStandardAttributes.json

```diff
@@ -14,12 +14,10 @@
             "version_added": null
           },
           "firefox": {
-            "version_added": "3",
-            "version_removed": "22"
```

Comment:
Wasn't it removed? What's the story with the existing data here?

Reply:
Short answer: mixin...

Long answer: This interface had a bit of a weird history.  In [SVG 1.1](https://www.w3.org/TR/SVG11/filters.html#InterfaceSVGFilterPrimitiveStandardAttributes), it was defined as its own standalone interface.  But then, in [SVG 2](https://drafts.fxtf.org/filter-effects-1/#InterfaceSVGFilterPrimitiveStandardAttributes), it had become a mixin.  The collector has custom IDL for the SVG 1.1 interface, so the results it is generating are based upon the assumption that it is an interface on its own, not a mixin.

I opened up this PR by copying the versions for the new BCD added into interfaces that implement the mixin.  I've also now removed the old interface definition.